appstudio-utils/util-scripts/verify-attestation-with-policy.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Verifies the given attestation passes the given rego policy
+# usage:
+#   verify-attestation-with-policy.sh <attestation file> <output file> <passed file>
+# where:
+#   <in-toto file> file containing the in-toto attestation to be verified
+#   <output file>  where to store the result in JSON format
+#   <passed file>  where to store the overall passing result
+set -euo pipefail
+
+in_toto="$1"
+output="$2"
+passed="$3"
+
+# An in-toto attestation is authenticated metadata about one or more software
+# artifacts, as per the SLSA Attestation Model:
+# https://github.com/slsa-framework/slsa/blob/main/controls/attestations.md
+# It is assumed at this point that the in-toto attestation has been fetched
+# securely and all signature checks have passed, e.g. via cosign. Therefore,
+# a policy is only applied to the embedded SLSA attestation.
+attestion=$(jq '.payload | @base64d | fromjson' ${in_toto})
+
+cd $(dirname $0)
+source lib/fetch.sh
+
+# TODO: If git clone fails, the task does not fail - fix that!
+./fetch-ec-policies.sh
+
+save-policy-config
+
+title Predicate
+# Show a shortened version of the attestions to avoid excessive logging
+echo -n $attestion | jq -r '.predicate = "..."'
+# Save the attestion for opa processing
+echo -n ${attestion} > $(json-data-file attestations)
```

Comment:
Same thought as in the ec-policies PR. Would there be multiple of them? Or at least, would the content of that file be an array? If not then maybe "attestation" is a better filename here.

Reply:
This is addressed now. `data/attestations/data.yaml` is now always a list of attestations.